src/inventory/inventory.py

```diff
@@ -19,33 +21,89 @@ def __init__ (self):
         else:
             self.__checkForUpdates()
 
-    # TODO... Implement
-    def add_card(self, card):
-        self.connection.commit()
-        return False
-
-    # TODO... Implement
-    def remove_card(self, card):
-        self.connection.commit()
-        return False
+   # TODO: Add count values to database to track cards contained in inventory
+    def addCardToInventory(self, card, to_add):
+        print('adding card')
+        placeholder = input('')
+        # self.connection.commit()
+        # total = card.count + to_add
+        # query = f"UPDATE 'MTG-Cards count' = {total} FROM WHERE id = {card.ID};"
+        # self.connection.execute(query)
+
+   # TODO: Add count values to database to track cards contained in inventory
+    def removeCardFromInventory(self, card, to_remove):
+        print('removing card')
+        placeholder = input('')
+        # self.connection.commit()
+        # total = card.count - to_remove
+        # if total < 0:
+        #     total = 0
+        # query = f"UPDATE 'MTG-Cards count' = {total} FROM WHERE id = {card.ID};"
+        # self.connection.execute(query)
+
+   # TODO: Add count values to database to track cards contained in inventory
+    def displayInventory():
+        print('Displaying inventory')
+        placeholder = input('')
 
     def close (self):
         self.connection.commit()
         self.connection.close()
 
-    # TODO, reimagine each of these find_cards_* methods, as filters methods.
-    # Provide them with the list that is to be filtered.
-    # This will allow complex chaining, of searches.
-    def find_cards_with_set(self,searched_setname):
-        '''Returns a list of Dataframes for each card in the ManaPool-Inventory.db matching the provided setname'''
-        return f"Stub, but here's {searched_setname}"
-        #return self.cursor.execute(f"SELECT * FROM 'MTG-Cards' WHERE set_name={searched_setname}'")
-
-    def find_cards_with_name(self, searched_cardname):
-        '''Returns a list of Dataframes for each card in the ManaPool-Inventory.db matching the provided card name'''
-        return f"Stub, but here's {searched_cardname}"
-        #return self.cursor.execute(f"SELECT * FROM 'MTG-Cards' WHERE name={searched_cardname}'")
-
+   # TODO: Add count values to database to track cards contained in inventory
+    def searchBySet(self, setname):
+        query = f"SELECT id, multiverse_ids, name, mana_cost, type_line, set_name, rarity FROM '{self.table_name}' WHERE set_name='{setname}'"
```

Comment:
Could improve the amount of results returned if we opt for checking for the input pattern, contrary to an exact match.

Reply:
I think we should talk about how we want to implement this in our next standup. I agree with you, but there are a number of ways to go about it.